src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -296,6 +296,45 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
     return this.hasRole(RoleTypes.APPLICANT, 1, CompareModes.EXACT)
   }
 
+  /** True when orgPerson applicant role. */
+  public isApplicant (orgPerson: OrgPersonIF): boolean {
+    return orgPerson?.roles.some(role => role.roleType === RoleTypes.APPLICANT)
+  }
+
+  get applicantPersons (): OrgPersonIF[] {
+    return this.getOrgPeople.filter(person =>
+      this.isApplicant(person) && this.isPartyTypePerson(person) && !this.wasRemoved(person)
+    )
+  }
+
+  get applicantOrgs (): OrgPersonIF[] {
+    return this.getOrgPeople.filter(person =>
+      this.isApplicant(person) && this.isPartyTypeOrg(person) && !this.wasRemoved(person)
+    )
+  }
+
+  get applicantPersonsRemoved (): OrgPersonIF[] {
```

Comment:
If you never use the list from here (or below), consider changing it to return a boolean.

See https://www.w3schools.com/jsref/jsref_some.asp

Reply:
Yes. I think I started out with a different thought but makes sense to change.

Reply:
Using `some` will be faster, since it will return as soon as it finds a match. Also, it already returns a boolean.

Reply:
Nice simplification 👍